tests/snapshots/framework-detection.test.js.md

```diff
@@ -66,77 +64,100 @@ Generated by [AVA](https://avajs.dev).
     ␊
     `
 
-## should report error if using static server and `command` is configured
+## should warn if using static server and `command` is configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Using simple static server because --dir flag was specified␊
+    ◈ Using simple static server because '--dir' flag was specified␊
+    ◈ Ignoring command setting since using a simple static server. Configure 'command' and 'targetPort' for a custom setup␊
     ◈ Running static server from "site-with-index-file/public"␊
-    ◈ "command" option cannot be used in conjunction with "dir" flag which is used to run a static server`
+    ␊
+    ◈ Static server listening to 88888␊
+    ␊
+       ┌──────────────────────────────────────────────────┐␊
+       │                                                  │␊
+       │   ◈ Server now ready on http://localhost:88888   │␊
+       │                                                  │␊
+       └──────────────────────────────────────────────────┘␊
+    ␊
+    `
 
-## should error if using static server and `targetPort` is configured
+## should warn if using static server and `targetPort` is configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Using simple static server because --dir flag was specified␊
+    ◈ Using simple static server because '--dir' flag was specified␊
+    ◈ Ignoring 'targetPort' setting since using a simple static server.␊
+    ◈ Use --staticServerPort or [dev.staticServerPort] to configure the static server port␊
     ◈ Running static server from "site-with-index-file/public"␊
-    ◈ "targetPort" option cannot be used in conjunction with "dir" flag which is used to run a static server`
+    ␊
+    ◈ Static server listening to 88888␊
+    ␊
+       ┌──────────────────────────────────────────────────┐␊
+       │                                                  │␊
+       │   ◈ Server now ready on http://localhost:88888   │␊
+       │                                                  │␊
+       └──────────────────────────────────────────────────┘␊
+    ␊
+    `
 
-## should error when `command` and `targetPort` when configured, but framework is not
+## should run `command` when both `command` and `targetPort` are configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ "framework" option must be set to "#custom" when specifying both "command" and "targetPort" options`
+    ◈ Starting Netlify Dev with custom config␊
+    hello␊
+    ◈ "echo hello" exited with code *. Shutting down Netlify Dev server`
 
-## should error when a specific framework is configured but not detected
+## should force a specific framework when configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Specified "framework" detector "create-react-app" did not pass requirements for your project`
+    ◈ Starting Netlify Dev with Create React App␊
```

Comment:
Expected change in behavior, if the user configures `framework = 'create-react-app'` we simply use it instead of running that detector and failing if it doesn't match